pkg/operator/configobservation/network/observe_network.go

```diff
@@ -101,3 +102,31 @@ func ObserveServicesSubnet(genericListers configobserver.Listers, recorder event
 
 	return out, errs
 }
+
+// ObserveExternalIPPolicy observes the network configuration and generates the
+// ExternalIPRanger admission controller accordingly.
+func ObserveExternalIPPolicy(genericListers configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (map[string]interface{}, []error) {
+	// set observed values
+	//  admission:
+	//    pluginConfig:
+	//      network.openshift.io/ExternalIPRanger:
+	//        configuration:
+	//          version: network.openshift.io/v1
+	//          kind: ExternalIPRangerAdmissionConfig
+	//          externalIPNetworkCIDRs: [...]
+	observedConfig := map[string]interface{}{}
+	configPath := []string{"admission", "pluginConfig", "network.openshift.io/ExternalIPRanger", "configuration"}
+
+	// For 4.1, we didn't expose the ExternalIP in the network spec so only need to block all IPs by default.
+
+	// Policy, synthesize config
+	// Simply by creating this configuration, the admission controller will
+	// be enabled and block all IPs by default.
+	admissionControllerConfig := unstructured.Unstructured{}
+	admissionControllerConfig.SetAPIVersion("network.openshift.io/v1")
+	admissionControllerConfig.SetKind("ExternalIPRangerAdmissionConfig")
+	unstructured.SetNestedField(admissionControllerConfig.Object, false, "allowIngressIP")
```

Comment:
`allowIngressIP: false` is missing in the comment above, isn't it?

Reply:
and `externalIPNetworkCIDRs` I don't see being set here.

Reply:
Added the comment for `allowIngressIP: false`.
The `externalIPNetworkCIDRs` is going to be an empty slice in this case since there is no property in 4.1 under https://github.com/openshift/api/blob/release-4.1/config/v1/types_network.go#L28 to translate this to.